.docker/Makefile

```diff
@@ -0,0 +1,17 @@
+VERSION := 0.1.0
+PREFIX  := xaptum/xtt-cpp
+
+.DEFAULT_GOAL := all
+
+all:
+	$(MAKE) pull || $(MAKE) release
+
+pull: Dockerfile
+	docker pull $(PREFIX):$(VERSION)
+
+build: Dockerfile
+	docker build --rm --force-rm -t $(PREFIX):$(VERSION) - < $<
+
+release: Dockerfile
```

Comment:
I don't think `all` will work as expected.  It is missing a `$(MAKE) build`, so the `release` call will fail. (This is probably  a bug on `enftun` too.)